### PR TITLE
New Port: Elec Dude

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Elec Dude

## Game Information
- **Title**: Elec Dude
- **URL**: https://store.steampowered.com/app/1907980/Elec_Dude/

### CFW Tests
CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

Architecture
[X] armHF
[X] Panfrost (Mainline)

Resolution Tests
Test all major resolutions:
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/elecdude/Elec Dude.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    if [ "$checksum" = "adc1dc81b4e8b1640f29714400333b62" ]; then
+        $ESUDO $controlfolder/xdelta3 -d -s gamedata/data.win -f ./patch/itch.xdelta gamedata/game.droid && \
+        rm gamedata/data.win
+    # Checksum for the Steam version
+    elif [ "$checksum" = "0e8fe70a1aca955a1db05a338f9ee6b7" ]; then
+        $ESUDO $controlfolder/xdelta3 -d -s gamedata/data.win -f ./patch/steam.xdelta gamedata/game.droid && \
+        rm gamedata/data.win
+    else
+        echo "Error: MD5 checksum of game.unx does not match any expected version."
+    fi
```
Comment (kloptops) [inline ports/elecdude/Elec Dude.sh:59]: Should the script exit here?
```suggestion
      exit 1
    fi
```

File: ports/elecdude/Elec Dude.sh [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+else    
+    echo "Error: Missing files in gamedata folder or game has been patched."
+fi
```
Comment (kloptops) [inline ports/elecdude/Elec Dude.sh:62]: And here:
```suggestion
   exit 1
fi
```

**Review by kloptops — CHANGES_REQUESTED** [review]
A few small changes in the bash script. 😁

File: ports/elecdude/Elec Dude.sh (hunk 1 above)
Comment (Fraxinus88) [reply ports/elecdude/Elec Dude.sh:59]: Shall input this.

File: ports/elecdude/Elec Dude.sh (hunk 2 above)
Comment (Fraxinus88) [reply ports/elecdude/Elec Dude.sh:62]: Leaving this out as it is not an actual error

**Review by Cebion — APPROVED** [review]

File: ports/elecdude/Elec Dude.sh (hunk 2 above)
Comment (kloptops) [reply ports/elecdude/Elec Dude.sh:63]: Excellent. :)

**Review by kloptops — APPROVED** [review]

**Pull request merged**